### ISSUE TITLE
Problem with aurelia/bootstrapper

### ISSUE THREAD

**Issue by ghost** [jdanyow/aurelia-breeze-northwind#9]
Cannot GET /dist/github:aurelia/bootstrapper@0.14.0.js

Cloned the project. installed on windows 10 machine.
npm install and all steps went smooth, no errors reported.

App starts up but spinner runs continuously. 
Console shows 404 and the above message.

Any help would be appreciated.

**Comment by johntom** [comment]
Hi,
I just did a new install on a windows 10 computer and had no issues.

``` shell
 npm install
 npm install -g jspm
```

> Taken from skeleton repo 
> **Note:** jspm queries GitHub to install semver packages, but GitHub has a rate limit on anonymous API requests. It is advised that you configure jspm with your GitHub credentials in order to avoid problems. You can do this by executing `jspm registry config github` and following the prompts. If you choose to authorize jspm by an access token instead of giving your password (see GitHub `Settings > Personal Access Tokens`), `public_repo` access for the token is required.
> 5. Install the client-side dependencies with jspm:

``` shell
  jspm install -y
```

HTH
John

**Comment by ghost** [comment]
Thanks for the reply John. Yes, I've done everything as described. I tried with jspm and jspm@beta. Git is set in my path. As I said before there are no errors on install. My other node sites work fine it's just this one that has the problem. Will keep at it.
Reading this. Not sure if it helps me though: https://github.com/aurelia/bootstrapper/issues/10

**Comment by JimmiV** [comment]
Same problem here.

I'm not able to run it on my Win10 machine. I ran others aurelia examples with success, but this one give me 2 errors while starting (F12 in Chrome):

1) GET http://localhost:9000/dist/github:aurelia/bootstrapper@0.14.0.js 404 (Not Found)
2) Uncaught (in promise) Error: XHR error (404 Not Found) loading http://localhost:9000/dist/github:aurelia/bootstrapper@0.14.0.js(…)

During nmp install I saw some buld errors, jspm install completed with no error!
I noticed a message about boostrapper fork installation bootstrapper@0.14.0 and bootstrapper@0.14.1.

Some hints?

TY!
